fearless/Common/DataProvider/PriceProviderFactory.swift

```diff
@@ -62,7 +62,7 @@ extension PriceProviderFactory: PriceProviderFactoryProtocol {
 
         let source = CoingeckoPricesSource(pricesIds: pricesIds)
 
-        let trigger: DataProviderEventTrigger = [.onNone]
+        let trigger: DataProviderEventTrigger = [.onAddObserver]
```

Comment:
check this

Reply:
Need implemented layer which the will work with all prices, and we be do refresh for priceProveder instead of create new provider